README.md

```diff
@@ -76,6 +76,25 @@ the relevant html reports, for instance:
     - file:///foo/project/build/reports/pmd/main4.html
 ```
 
+It's possible to specify a custom renderer for the report urls in the logs via the `logs` extension. This can be useful in CI
+ environments, where the local paths are not reachable directly. For instance the snippet below will replace the base url with
+ one of your choice:
+```gradle
+staticAnalysis {
+    ...
+    logs {
+        reportBaseUrl "http://ci.mycompany.com/job/myproject/ws/app/build/reports"
```

Comment:
I was just worried they don't match up because in my mind we previously only talked about the base url as the workspace. If the reports folder is the same for all tools and that's the base I'm super happy with this example.